detection_rules/packaging.py

```diff
@@ -135,6 +136,22 @@ def _package_notice_file(save_dir):
             lines = lines + commented_notice + [' */', '']
             f.write('\n'.join(lines))
 
+    def _package_index_file(self, save_dir):
+        """Convert and save index file with package."""
+        e_license = ['/*', '\n'.join(' * ' + line.lstrip('# ') for line in LICENSE_HEADER.splitlines()), ' */', '']
+        comments = [
+            '// Auto generated file from scripts/regen_prepackage_rules_index.sh',
+            '// Do not hand edit. Run that script to regenerate package information instead',
```

Comment:
Should we point to this repository instead?
Or explain both?

Reply:
I'll add both